.flake8

```diff
@@ -3,7 +3,8 @@ max-line-length = 88
 # E203: whitespace before ‘,’, ‘;’, or ‘:’
 # E501: line length
 # W503: line break after binary operator
-ignore = E203, E501, W503
+ignore = E203, E501, W503, D, DAR
+extend-select = B902, B903, B904
```

Comment:
this flake8 config keeps creeping back into the PR! I'll wait to see if there are any more edits and then remove it.